plugins/ltac2/tac2ffi.mli

```diff
@@ -130,6 +130,10 @@ val of_sort : ESorts.t -> valexpr
 val to_sort : valexpr -> ESorts.t
 val sort : ESorts.t repr
 
+val of_rewstrategy : Rewrite.strategy -> valexpr
+val to_rewstrategy : valexpr -> Rewrite.strategy
+val rewstrategy : Rewrite.strategy repr
```

Comment:
Is this okay or should rather I store strategy AST here?

Reply:
looks ok